webapp/backend/.gitignore

```diff
@@ -0,0 +1,8 @@
+// node modules
```

Comment:
comments in .gitignore should begin with `#` instead of `//`

Reply:
Great. Thanks for the comment. Implemented change.